luigi/contrib/hadoop.py

```diff
@@ -971,7 +971,7 @@ def dump(self, directory=''):
             if self.__module__ == '__main__':
                 d = pickle.dumps(self)
                 module_name = os.path.basename(sys.argv[0]).rsplit('.', 1)[0]
-                d = d.replace(b'(c__main__', "(c" + module_name)
+                d = d.replace(b'c__main__', b'c' + module_name.encode('ascii'))
```

Comment:
I think it's better we keep `(` and encode with default encoding (utf-8).

Reply:
I've made the revision according to your suggestion. New submission is committed. 

Thank you